sonar-db/src/main/java/org/sonar/db/issue/IssueDto.java

```diff
@@ -158,8 +158,8 @@ public static IssueDto toDtoForUpdate(DefaultIssue issue, long now) {
       .setLine(issue.line())
       .setLocations((DbIssues.Locations) issue.getLocations())
       .setMessage(issue.message())
-      .setEffortToFix(issue.effortToFix())
-      .setDebt(issue.debtInMinutes())
+      .setGap(issue.effortToFix())
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "effortToFix"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)